src/material/stepper/stepper.md

```diff
@@ -222,3 +222,19 @@ is given `role="tab"`, and the content that can be expanded upon selection is gi
 step content is automatically set based on step selection change.
 
 The stepper and each step should be given a meaningful label via `aria-label` or `aria-labelledby`.
+
+When a stepper is used on a smaller screen, verical steppers are preferred over
+horizontal steppers because the text in the header will be more visible.
+
+#### Forms
+Steppers often have forms inside of it. It's best practice for form fields to
```

Comment:
inside of it -> inside of them

Reply:
```suggestion
Steppers often contain forms and form controls. If validation errors inside of a stepper's form prevents moving to another step, make sure that your form controls communicate error messages to assistive technology. This helps the user know why they can't advance to another step. You can accomplish this by using `<mat-error>` with `<mat-form-field>`, or by using an ARIA live region. 
```
* Changes for passive voice and grammar 
* Clarify that `<mat-error>` is one possible way to communicate updates